portfolio/src/main/webapp/style.css

```diff
@@ -9,29 +9,67 @@
   float: left;
 }
 #navbar li a {
+  color: #000000;
   display: block;
-  color: black;
-  text-align: center;
   padding: 14px 16px;
+  text-align: center;
   text-decoration: none;
 }
 #navbar li a:hover {
-  background-color: lightcoral;
+  background-color: #006c84;
+  color: #ffffff;
 }
 
 #content {
-  margin: auto;
   display: block;
+  margin: auto;
   text-align: center;
 }
 
 #greeting-container {
   margin-top: 30px;
-  color: blanchedalmond;
 }
 
 #homelinks img {
+  padding: 10px;
   width: 50px;
+}
+
+#projects-content {
+  left: 10%;
+  margin-right: auto;
+  position: relative;
+}
+#projects-content .project-block {
+  border: 1mm double #006c84;
+  margin: 10px;
+  padding: 15px;
+  width: 60%;
+}
+
+#blog-content {
+  margin: auto;
+  text-align: center;
+  width: 80%;
+}
+#blog-content #blog-posts {
+  display: grid;
+  grid-template-columns: auto auto auto;
+  grid-gap: 100px 100px;
   padding: 10px;
 }
+#blog-content .post {
+  border: 2mm double #006c84;
+  padding: 50px;
+}
+#blog-content a {
+  text-decoration: none;
+}
+#blog-content a:visited {
+  color: #000000;
+}
 
+#post1 img {
+  max-width: 100%;
+  max-height: 70vh;
+}
```

Comment:
With the growing css file since most of them seem to be divided by IDs, you could consider splitting the css files into the page/project-specific css (one css file for each of the pages), with a single common css that runs across all your separate pages as well.

This will make it easier to have the independent PRs, as well as make the css cleaner and more structured, imo.

Reply:
Made a css file for just the posts folder!